### PR TITLE
288 Add logic to process custom webhooks

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: posttowebhooks.go [hunk 1]
```diff
@@ -115,6 +115,10 @@ func isNotificationInTopic(msg []byte, topics []string) bool {
 func preparePayloadFromTemplate(templatetext string, msg []byte) *strings.Reader {
 	var buf1 bytes.Buffer
 	var postbody Payload
+	if len(templatetext) == 0 {
+		Log.Printf("Empty Template. message to post: %s", string(msg))
+		return strings.NewReader(string(msg))
```
Comment (ash3rz) [inline posttowebhooks.go]: Minor, but should `string(msg)` be put in a var since we're using that twice?

**Review by ash3rz — APPROVED** [review]
Not familiar with Go or the code, but seems pretty straightforward 🚢

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by psarando** [comment]
This PR could use a more descriptive title 😉

**Comment by sri2k1us** [comment]
> This PR could use a more descriptive title 😉

Wonder why github didn't auto populate the title. I blame github 😉

**Comment by psarando** [comment]
> Wonder why github didn't auto populate the title. I blame github 😉

😆 Ahh, because it has 2 commits. It will auto-populate when there's only 1 commit.

**Comment by ianmcorvidae** [comment]
Yeah, it just uses the branch name. I get a lot of "Core <number>" prs with `core-<number>` branches

**Pull request merged**